SmartTutor/KeycloakAuth/AllowedKeycloakRole.cs

```diff
@@ -0,0 +1,15 @@
+﻿using System;
+using Microsoft.AspNetCore.Authorization;
+
+namespace SmartTutor.KeycloakAuth
+{
+    public class AllowedKeycloakRole : IAuthorizationRequirement
+    {
+        public AllowedKeycloakRole(string role)
+        {
+            AllowedRole = role;
+        }
+
+        public string AllowedRole { get; }
```

Comment:
Will this be the code from which we build our RBAC model? We should explore how RBAC ties in with keycloak - does the role-permissions mapping need to be handled on our side or on Keycloak's side.

Reply:
We don't have any RBAC model in backend for now. Role is claimed from JWT that is gathered from Keycloak (Role system is managed on Keycloak side). On the backend side we just configure policy for the role that comes from Keycloak.